homeassistant/components/voice_assistant/__init__.py

```diff
@@ -31,8 +32,10 @@
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up Voice Assistant integration."""
-    hass.data[DOMAIN] = {}
+    pipeline_store = PipelineStore(hass)
+    await pipeline_store.async_load()
```

Comment:
can/should we delay this until the first request for a pipeline comes in?

Reply:
Sure, at the cost of all accesses to the store being coroutine functions. Is that OK?

Reply:
This is no longer relevant since we use `StorageCollectionWebsocket`